tests/ext/dd_trace_method.phpt

```diff
@@ -0,0 +1,174 @@
+--TEST--
+dd_trace_method() can trace with internal spans
+--FILE--
+<?php
+use DDTrace\SpanData;
+
+class Test
+{
+    public function testFoo()
+    {
+        echo "Test::testFoo()\n";
+    }
+}
+
+class TestService
+{
+    public function testServiceFoo()
+    {
+        echo "TestService::testServiceFoo()\n";
+    }
+}
+
+class Foo
+{
+    public function bar($thoughts, array $bar = [])
+    {
+        echo "Foo::bar()\n";
+        return [
+            'thoughts' => $thoughts,
+            'first' => isset($bar[0]) ? $bar[0] : '(none)',
+            'rand' => mt_rand(42, 999)
+        ];
+    }
+}
+
+var_dump(dd_trace_method('Test', 'testFoo', function (SpanData $span) {
```

Comment:
I like the distinction between dd_trace function and method 👍 Its a good change :)